edge-api-routes/authed-proxy/tsconfig.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we need anything here?

Reply:
No, but right now it isn't working without it (All your examples are broken). I'm assured it'll be fixed by EOD.